Resources/Prototypes/Entities/Objects/Misc/rubber_stamp.yml

```
@@ -245,7 +245,7 @@
   - type: Stamp
     stampedName: stamp-component-stamped-name-approved
     stampedColor: "#00be00"
-    stampState: "paper_stamp-approve" # Corvax-IAA: paper_stamp-ok state is IAA!
+    stampState: "paper_stamp-approve" # Corvax
```

```suggestion
    stampState: "paper_stamp-ok"
```

но вместе с этим переименуй paper_stamp-approve в paper_stamp-ok и тот paper_stamp-ok который есть сейчас нужно удалить (почему то он использовался для печати авд, хотя можно было сделать абсолютно новый спрайт с новым именем, да морти, я смотрю на тебя)